### PR TITLE
Feature/account views

### PR DESCRIPTION
**Pull request by ezekielaquino**
Establishes account views:
Needs further interactions and further component visual polish (shared with other views)

### `/account` Dashboard
- Account dashboard Basic layout and base components (some components have placeholders for shared components with other views)
- Relevant buttons link to `/account/activity` and `/account/history`. Note: modal interactions are not yet in place
- "Mock" data

### `/account/activity` Account activity
- Table component with mock data
- Breadcrumb component and link to account page

### `/account/history` Account / stake history
- Same as account activity view
- Table component with placeholder data

### CODE REVIEW

File: ts/components/ui/breadcrumb.tsx [hunk 1]
```diff
@@ -0,0 +1,86 @@
...
+            </ButtonBack>
+
+            <p>
+                {_.map(crumbs, crumb => {
+                    const Component = crumb.url ? ReactRouterLink : 'span';
+
+                    return (
+                        <Component
+                            key={crumb.url}
+                            to={crumb.url}
```
Comment (kimpers) [inline ts/components/ui/breadcrumb.tsx]: won't this pass an invalid prop `to` if `Component` is a `span`?

File: ts/components/ui/panel_header.tsx [hunk 2]
```diff
@@ -0,0 +1,82 @@
+import * as React from 'react';
+import styled from 'styled-components';
+
+import { Heading, Paragraph } from 'ts/components/text';
+import { colors } from 'ts/style/colors';
+
+interface PanelHeaderProps {
+    avatarSrc?: string;
+    avatarComponent?: Node;
+    title: string | Node;
```
Comment (kimpers) [inline ts/components/ui/panel_header.tsx]: Should these not be `React.ReactNode`?

File: ts/constants/utilities.tsx [hunk 3]
```diff
@@ -20,3 +20,13 @@ export const getCSSPadding = (value: number | Array<string | number> = 0): strin
         return `${value}px`;
     }
 };
+
+export const truncateStringPortion = (value: string, beginCharCount: number = 3, endCharCount: number = 3, dotCount: number = 3) => {
```
Comment (kimpers) [inline ts/constants/utilities.tsx]: There is already a `getAddressBeginAndEnd in `ts/constants/utils` might be better to extended it with additional optional params

File: ts/components/ui/breadcrumb.tsx (hunk 1 above)
Comment (ezekielaquino) [reply ts/components/ui/breadcrumb.tsx]: Refactored in 5e75e633ec42dffbef68bbe924fcf2271698724a

File: ts/components/ui/panel_header.tsx (hunk 2 above)
Comment (ezekielaquino) [reply ts/components/ui/panel_header.tsx]: Indeed! This has been changed in f84cd8fb02a55d4495fc81312b6b3fd200b23344

File: ts/constants/utilities.tsx (hunk 3 above)
Comment (ezekielaquino) [reply ts/constants/utilities.tsx]: Removed the redundant function and extended the previous method in utils in a7832e8dc04b795294df7df319677d808c05b15f

**Review by kimpers — APPROVED** [review]

**Pull request merged**